plugins/CoreAdminHome/CustomLogo.php

```diff
@@ -178,17 +179,55 @@ public function copyUploadedLogoToFilesystem()
     {
         $uploadFieldName = 'customLogo';
 
-        $success = $this->uploadImage($uploadFieldName, self::LOGO_SMALL_HEIGHT, $this->getPathUserLogoSmall());
-        $success = $success && $this->uploadImage($uploadFieldName, self::LOGO_HEIGHT, $this->getPathUserLogo());
+        $rootPath = Filesystem::getPathToPiwikRoot();
+        $smallLogoUserPath = $this->getPathUserLogoSmall();
+        $logoUserPath = $this->getPathUserLogo();
 
-        return $success;
+        $success = $this->uploadImage($uploadFieldName, self::LOGO_SMALL_HEIGHT, $smallLogoUserPath);
+        if (!$success) {
+            return false;
+        }
+
+        /**
+         * Triggered when a user uploads a custom logo. This event is triggered twice,
+         * once for the small logo file and once for the normal sized logo.
+         *
+         * @param string $absolutePath The absolute path to the logo file on the Piwik server.
+         * @param bool $isSmall If true, the path is for the small logo image, otherwise the
+         *                      full image.
+         */
+        Piwik::postEvent('CoreAdminHome.customLogoChanged', [$rootPath . '/' . $smallLogoUserPath, $isSmall = true]);
```

Comment:
Looks good to me. Maybe, instead of `isSmall:true|false` could post something like `size=small`? Would just allow us to more easily post more events in the future in case we have other sizes.

Reply:
I just see below the favicon event... could event have like `size=small|large|favicon` maybe?